src/config.rs

```diff
@@ -421,6 +422,9 @@ pub struct ClusterConfig {
     ))]
     capella_org: Option<String>,
 
+    #[serde(default)]
```

Comment:
I'm assuming this just means that when deserialising it will default to specifying it as an empty string?

Reply:
If unspecified, serde will deserialise it into a `None` (And `None` is serialised into a JSON `null`)

Reply:
Thanks @Matt-Woz 
@Westwooo I don't really get why we want this to be null when deserialised, but others, such as cluster_type would instead presumably just not have a value?

Reply:
Good point. cluster_type and project would both deserialise into None when missing since they are options, so have removed the unnecessary `#[serde(default)]` attribute, also have removed it from capella_organization.